### PR TITLE
Nav avatar

### PR DESCRIPTION
**Pull request by traceypooh**
Removes code no longer used/wanted that was adding load and creating a bug w/ logged-in users w/ custom avatar imagery. 
Inadvertently, it also has some commits to make the `ia-topnav` be able to be used in a dynamic review app fashion via https://coderunr.com

> What does this PR achieve? Is it a feature/hotfix/refactor? Does it close an Issue? If so, use `Closes #`.

part of WEBDEV-5864

> What should be noted about the implementation?

ensure review app top page nav looks normal -- though LOL there is a *huge chance* you'll see grayed out generic image (The Bug) until petabox parallel MR is merged *and live*

> What steps should the reviewer take to verify this PR resolves the issue?

**Evidence**

> If this PR touches UI, please post evidence (screenshot) of it behaving correctly.

### CODE REVIEW

File: packages/ia-topnav/www/index.html [hunk 1]
```diff
@@ -1,5 +1,15 @@
 <link rel="stylesheet" href="https://archive.org/includes/build/css/archive.min.css">
 
+<script type="importmap">
+  {
+    "imports": {
+      "@internetarchive/ia-wayback-searchx": "https://esm.archive.org/@internetarchive/ia-wayback-search",
```
Comment (bfalling) [inline packages/ia-topnav/www/index.html:6]: QUESTION: What is this `ia-wayback-searchx`? Seems like an effective commenting out? If so, can we just remove this line?

**Review by bfalling — COMMENTED** [review]
Seems reasonable. How to test it?

File: packages/ia-topnav/www/index.html (hunk 1 above)
Comment (traceypooh) [reply packages/ia-topnav/www/index.html]: yes, sorry, an oversight indeed!  i removed it.  (artifact while testing out import maps)

**Comment by traceypooh** [comment]
testing can be done via coderunr here:
https://iaux-nav-avatar.code.archive.org/ia-topnav/

**Pull request merged**